### PR TITLE
Add role assignment public ip & api_server_authorized_ip_ranges

### PR DESCRIPTION
**Pull request by gustavoborragan**

### CODE REVIEW

**Comment by gustavoborragan** [comment]
https://github.com/councilbox/claims/pull/797

File: modules/azure-aks/role_assignment.tf [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/role_assignment
+resource "azurerm_role_assignment" "role_assignment_network_contributor_over_public_ip_aks_predev" {
```
Comment (jcframil) [inline modules/azure-aks/role_assignment.tf:2]: whay named predev?

**Review by jcframil — CHANGES_REQUESTED** [review]
check commented resource naming

File: modules/azure-aks/role_assignment.tf (hunk 1 above)
Comment (gustavoborragan) [reply modules/azure-aks/role_assignment.tf:2]: Changed

**Pull request merged**